### ISSUE TITLE
Add term and structure for transverse terminal limb defect

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10479]
**Preferred term label:**
Transverse terminal limb defect

**Synonyms**
Transverse terminal limb reduction

**Definition (free text, please give PubMed ID)**
A congenital anomaly that appears as an “amputation” at different levels of the extremity. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
PMID: 34263545

**Parent term (use hpo.jax.org/app)**
Abnormality of limbs HP:0040064

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adams-Oliver syndrome (OMIM100300)

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

-------------------------------------------------------------------------------------------
This term would be the parent term of transverse terminal upper limb defect and transverse terminal lower limb defect. These terms will be then splitted into the level of 'amputation' based on the classification system of Frantz and O’Rahilly (Frantz, C. H., and Ronan O'Rahilly, Congenital skeletal limb deficiencies, J. Bone & Joint Surg., Boston, 43A:1202-1224, December 1961.) I sketched the tree structure below:

![20240422_125800](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/c97644cb-08f4-48fd-bf99-0b3cb04f72f4)

**Comment by pnrobinson** [comment]
@lisannesw 
adding term for now, in the next round we can add children
Adding new term: 
- ID: HP:6000818
- label: Transverse terminal limb defect
- parents: [Abnormality of limbs (HP:0040064)](https://hpo.jax.org/app/browse/term/HP:0040064)
- pmid: PMID:34263545
- def: A congenital anomaly that appears as an amputation at different levels of the extremity. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10479
- synonyms: Transverse terminal limb reduction(NONE)

**Comment by pnrobinson** [comment]
THanks, added as suggested!
